### PR TITLE
Adds `dctl contract init`

### PR DESCRIPTION
**Pull request by deanshelton913**
# Adds Smart Contract Test Functionality
There is currently no way to stub heap data within smart contracts for testing purposes.

The goal of this PR is to:

- [x] Add a `dctl contract init` 
  - Create a test directory to store .env file
  - Create a directory used to fake the heap
- [x] Add a `dctl contract test` 
  - Run a docker file while mounting the env and fake heap.
  - Provide an intuitive testing "event" which fakes the input a contract can expect.

# Companion PR required in SDKs 
This will require a companion PR within the SDK repositories to allow for stubbing of heap data.

### CODE REVIEW

**Review by rdaquilante — APPROVED** [review]

File: src/dctl-contract-test.js [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+function createDockerNetwork(networkName) {
+  shell.exec(`docker network create ${networkName}`, { silent: true });
+}
+
+async function getConfig(testRoot) {
+  return JSON.parse(await fs.promises.readFile(path.join(testRoot, 'config.json'), 'utf-8'));
+}
+
+async function runContract(image, payload, network, startCommand, localEnv, localSecrets) {
+  const arrOfSecretFiles = await fs.promises.readdir('./test/secrets');
```
Comment (cheeseandcereal) [inline src/dctl-contract-test.js]: should use a cross-platform way to form this path. I.e. windows uses '\', not '/'

Comment (deanshelton913) [reply src/dctl-contract-test.js]: yupyup. fixed

File: webserver/dragonchain/start.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+def start() -> None:
+    """
+    Ran by the webserver before it boots
+    """
+    _log.info("Booting flask")
+    app = flask.Flask(__name__)
+    route(app)
+    app.run(host='0.0.0.0', port=8080)
+    _log.info('killing server.')
+    # do stuff
```
Comment (wolfpack94) [inline webserver/dragonchain/start.py]: Does this need more stuff?

Comment (deanshelton913) [reply webserver/dragonchain/start.py]: nah

**Review by DRigby26 — APPROVED** [review]

**Pull request merged**